docs/content/foundations/layout-positioning/float.mdx

```diff
@@ -8,9 +8,7 @@ utility_modules:
   - name: Float
     description: Places an element on the left or right side of its container, allowing text and inline elements to wrap around it.
     utility_module_name: 'float'
-    utility_class_intro: |
-      The float utility classes have not been modified from the U.S. Web Design System (USWDS). Refer to the [USWDS Float utility page](https://designsystem.digital.gov/utilities/float/) for a complete listing of classes.
-    utility_class_display_component: 'FloatDemonstration'
+    utility_class_display_component: 'TestUtilityClassTable'
```

Comment:
@karlikpj why are you reverting this back to before there was a demonstration class/

Reply:
merge issue / fixing